ci-operator/jobs/openshift/router/openshift-router-master-presubmits.yaml

```diff
@@ -66,7 +66,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e,?($|\s.*)
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
Is this change intentional?  We had `always_run: true` even before #12570.

Reply:
`always_run: true` was set for `pull-ci-openshift-router-master-e2e-metal-ipi`, which has been renamed to ` pull-ci-openshift-router-master-e2e-metal-ipi-ovn-ipv6`. This diff sets `always_run: false` for the new dualstack job (the prior metal ipi job was v6, not dualstack, so I figured we don't want the new dualstack job to always run).